<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1436</width>
    <height>847</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="notification_label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>0</y>
      <width>1280</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>48</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Notification Label</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="on_pushButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>102</width>
      <height>102</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="camera1_label">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>210</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Camera 1: Top View</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="camera2_label">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>210</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Camera 2: Left View</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="camera3_label">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>210</y>
      <width>251</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Camera 3: Right View</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="camera1">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>270</y>
      <width>381</width>
      <height>271</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="logo_label">
    <property name="geometry">
     <rect>
      <x>1300</x>
      <y>670</y>
      <width>131</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string> Discrete Output</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>190</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Control</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>100</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>High Performance (YoloV8)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Low  Performance (Tiny Yolo)</string>
    </property>
   </widget>
   <widget class="QLabel" name="camera2">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>270</y>
      <width>381</width>
      <height>271</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="camera3">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>270</y>
      <width>381</width>
      <height>271</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1436</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
